static/scripts/set-tab-language.js

```diff
@@ -13,9 +13,9 @@
       return entries;
     }, {});
 
-  if (entries.language === "js") {
+  if (entries.lang === "js") {
     window.localStorage.setItem("docusaurus.tab.language", "js");
-  } else if (entries.language === "ts") {
+  } else if (entries.lang === "ts") {
     window.localStorage.setItem("docusaurus.tab.language", "ts");
   }
```

Comment:
If it's `js` or `ts`, we also want to set `docusaurus.tab.site-lang` to `typescript`